modules/templates-uploading.adoc

```diff
@@ -6,18 +6,20 @@
 [id="templates-uploading_{context}"]
 = Uploading a template
 
-If you have a JSON or YAML file that defines a template, for example as seen in this example, you can upload the template to projects using the CLI. This saves the template to the project for repeated use by any user with appropriate access to that project. Instructions on writing your own templates are provided later in this topic.
+If you have a JSON or YAML file that defines a template, you can upload the template to projects using the CLI. This saves the template to the project for repeated use by any user with appropriate access to that project. Instructions on writing your own templates are provided later in this topic.
```

Comment:
I believe that you generally want to use "about" rather than "on" as it has greater clarity, especially when translated: "Instructions on writing..." to "Instructions about writing...
""